sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/AvroGenericRecordToStorageApiProto.java

```diff
@@ -213,7 +281,7 @@ public static DynamicMessage messageFromGenericRecord(
     return builder.build();
   }
 
-  private static TableFieldSchema fieldDescriptorFromAvroField(Schema.Field field) {
+  private static TableFieldSchema fieldDescriptorFromAvroField(org.apache.avro.Schema.Field field) {
```

Comment:
fully qualified name not necessary here? same below

Reply:
I think I got asked by the compiler. Will revert